src/Coccinelle-for-Rust.md

```diff
@@ -43,6 +43,11 @@ flow is not yet supported.
 
 [A recent talk about Coccinelle for Rust](https://gitlab.inria.fr/coccinelle/coccinelleforrust/-/blob/main/talks/rfl.pdf)
 
+## Support
+
+We would like to thank [Collabora](https://www.collabora.com/) for their support for
+the development of Coccinelle for Rust.
```

Comment:
```suggestion
We would like to thank [Collabora](https://www.collabora.com) for supporting
the development of Coccinelle for Rust.
```

Reply:
Thank you!

Reply:
Since you were here, if you want this suggestion to be applied, I can do it before publishing.

Reply:
(And you are welcome! :)

Reply:
Yes, please. That would be great!
Thank you!

Reply:
It is now live -- please take a look. Thanks!

Reply:
Its is perfect. Thank you!